src/vs/base/common/labels.ts

```diff
@@ -434,6 +434,10 @@ export function unmnemonicLabel(label: string): string {
 	return label.replace(/&/g, '&&');
 }
 
+export function removeMnemonic(label: string): string {
```

Comment:
I am curious how this was not an issue before and what lead to this issue?

Reply:
@alexr00 I think this is what you could use:

https://github.com/microsoft/vscode/blob/ec82ef20706a4dc572812b288cb8c10f1090fc32/src/vs/base/common/labels.ts#L415-L431